pegs_common_utils.R

```diff
@@ -262,63 +262,374 @@ prepare_ukb_phenotype <- function(ukb_data, phenotype) {
 # Returns:-
 # A dataframe (epr_number = participant ID, Y = phenotype). All participants to
 # be excluded have their phenotype marked as NA.
-prepare_pegs_phenotype <- function(pegs_data, phenotype) {
+prepare_pegs_phenotype <- function(pegs_data, 
+                                   phenotype = c("lower_gi_polyps",'fibroids',
```

Comment:
all phenotype names should be lower case unless they are acronyms. Also, sort them alphabetically to aid readability.
I think shortening the names is not helpful unless they are too long - example:  "Bclots" is better read if fully spelled out "blood_clots".